prow/plugins/wip/wip-label.go

```diff
@@ -49,7 +50,15 @@ type event struct {
 }
 
 func init() {
-	plugins.RegisterPullRequestHandler(pluginName, handlePullRequest, nil)
+	plugins.RegisterPullRequestHandler(pluginName, handlePullRequest, helpProvider)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	// Only the Description field is specified because this plugin is not triggered with commands and is not configurable.
+	return &pluginhelp.PluginHelp{
+			Description: "The wip (Work In Progress) plugin applies the '" + label + "' label to pull requests that contain 'WIP' in their title.",
```

Comment:
Worth noting that most likely this label will block merges (assuming it's included in the tide query)?

Reply:
Also these are PRs that start with WIP, not just contain it. The regex is ugly though, not worth showing.

Reply:
Also removes the label when title changes

Reply:
Done.